routes/users.js

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable prefer-destructuring */
+/* eslint-disable no-console */
+const express = require('express');
+const crypto = require('crypto');
```

Comment:
I believe Bruno suggested that we use bcrypt. But if crypto works fine I guess just leave it. Just make sure all sensitive hashing information is in the .env so not on our public github repo

Reply:
Will do.